packages/rum-core/src/domain/view/trackViews.ts

```diff
@@ -202,6 +211,8 @@ function newView(
   let documentVersion = 0
   let endClocks: ClocksState | undefined
   const location = shallowClone(initialLocation)
+  const viewCustomerDataManager = createCustomerDataTrackerManager()
+  const contextManager = createContextManager(viewCustomerDataManager.getOrCreateTracker(CustomerDataType.View))
```

Comment:
**💬 suggestion:** ‏you should not instanciate `createCustomerDataTrackerManager()` here. Either:
* don't use a `CustomerDataTracker` for the View context (for now). For that, you'll need to adjust `createContextManager` to make the argument optional.
* pass the manager [we got in `startRum`](https://github.com/DataDog/browser-sdk/blob/6682216f7fb02574b66dd03c1e8966c9504a1b3c/packages/rum-core/src/boot/startRum.ts#L62) down to `trackViews`.

I'm enclined to do the former. We can discuss about it if you want.